.github/workflows/update_iers.yml

```
@@ -0,0 +1,36 @@
+name: Auto-update IERS tables
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+    jobs:
```

Accidental indentation?
```suggestion
jobs:
```